src/homepageExperience/containers/CliWizard.tsx

```diff
@@ -28,6 +28,8 @@ import {event} from 'src/cloud/utils/reporting'
 import {HOMEPAGE_NAVIGATION_STEPS_CLI} from 'src/homepageExperience/utils'
 import RateLimitAlert from 'src/cloud/components/RateLimitAlert'
 
+export const onboardingName = `InfluxDB CLI`
```

Comment:
One thing to be aware of, having a variable in the middle of a string makes global searching for a string a little more difficult, so if you're trying to find all places where the string `the Influx CLI` is, it might make it a little bit harder.